packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -77,6 +92,44 @@ export interface LogConfiguration {
   readonly secretOptions?: ExposedSecret[];
 }
 
+
+/**
+ * Fargate platform configuration
+ */
+export interface FargatePlatformConfiguration {
+  /**
+   * Fargate platform version
+   */
+  readonly platformVersion: ecs.FargatePlatformVersion
+}
+
+/**
+ * Whether or not to assign a public IP to the job
+ */
+export enum AssignPublicIp {
```

Comment:
This seems funky, can we not just convert all of this to a boolean?

Reply:
Yes